kubernetes_deploy/staging/migrate-db-job.yml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api-migrate-db
+spec:
+  ttlSecondsAfterFinished: 120
+  template:
+    metadata:
+      name: laa-legal-adviser-api-migrate-db
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: app
+        image: "<to be set by deploy_to_kubernetes>"
+        command: ["python", "manage.py", "migrate", "--noinput"]
+        env:
+        - name: SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: secret-key
+              key: SECRET_KEY
```

Comment:
_not a blocker_
Do we need the secret key for the migration?

Reply:
Good spot. I'll check when I revisit this work to make the production setup work